README.md

```diff
@@ -106,7 +106,7 @@ Please read **src/md-datatable.interfaces.ts** for details about the payload of
 To add `ng2-md-datatable` to your Material 2 theming file:
 
 ```scss
-@import '~ng2-md-datatable/datatable-theme';
```

Comment:
Normally with SASS what you can do is idiomatically:
- name your imports with a leading underscore (_foo.scss)
- call them "without" this leading underscore (foo.scss)
- with most loaders, omit file extension when called from another SASS file (foo).